renderer/actions/EditorActions.ts

```diff
@@ -47,3 +47,13 @@ export const downloadCode: EditorActions['downloadCode'] = () => ({
 export const uploadCode: EditorActions['uploadCode'] = () => ({
   type: consts.EditorActionsTypes.UPLOAD_CODE,
 });
+
+export const updateBitmap: EditorActions['updateBitmap'] = (bitmap: number) => ({
+  type: consts.EditorActionsTypes.UPDATE_BITMAP,
+  bitmap,
+})
+
+export const updateKeyboardToggle: EditorActions['updateKeyboardToggle'] = (keyboardToggle: boolean) => ({
+  type: consts.EditorActionsTypes.UPDATE_KEYBOARD_TOGGLE,
+  keyboardToggle,
```

Comment:
Let's rename `keyboardToggle` to `isKeyboardToggled` just to be more explicit about whether the variable is a boolean.

Reply:
@uthmanmomen13  Also rename `updateKeyboardToggle` to `updateIsKeyboardToggled`